address-controller/src/main/resources/swagger.json

```diff
@@ -0,0 +1,640 @@
+                }
+            }
+        },
+        "/addressses/{addressSpace}/{address}": {
+            "get": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Get Address by name",
+                "description": "Returns a single Address",
+                "operationId": "getAddress",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "address",
+                        "in": "path",
+                        "description": "Name of Address to return",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "successful operation",
+                        "schema": {
+                            "$ref": "#/definitions/Address"
+                        }
+                    },
+                    "404": {
+                        "description": "Address or AddressSpace not found"
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "addresses"
+                ],
+                "summary": "Delete an Address",
+                "description": "",
+                "operationId": "deleteAddress",
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "addressSpace",
+                        "in": "path",
+                        "description": "Name of AddressSpace",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "address",
+                        "in": "path",
+                        "description": "Name of Address to delete",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Deletion successful; List of remaining Addresses is returned.",
+                        "schema": {
+                            "$ref": "#/definitions/AddressList"
+                        }
+                    },
+                    "404": {
+                        "description": "Address or AddressSpace not found"
+                    }
+                }
+            }
+        }
+    },
+    "securityDefinitions": {
```

Comment:
Is there a purpose for the meta-data.  I presume that the swagger library can ingest this data and spit out a client?  Or is it mostly for documentation purposes?

Reply:
Forgot to remove this part. Will need to be re-added later with correct information. I believe it's used both for documentation and generating clients.